### PR TITLE
issue-3265:-HelmServiceInstallIT test case failure fix

### LINKED ISSUE #3265: HelmServiceInstallIT is failing on Windows
### Component

JKube Kit

### Description

Related to https://github.com/eclipse-jkube/jkube/issues/1338

:warning: A windows machine is required to reproduce and fix this issue

HelmServiceInstallIT is failing on windows with this error message:

```
Error:  Failures: 
Error:    HelmServiceInstallIT.whenDependencyReferencedDoesNotExist_thenThrowException:183 
Expecting message to be:
  "An error occurred while updating chart dependencies: directory C:\Users\RUNNER~1\AppData\Local\Temp\junit6642167052648161605/the-dependency not found"
but was:
  "An error occurred while updating chart dependencies: directory C:\Users\RUNNER~1\AppData\Local\Temp\junit6642167052648161605\the-dependency not found"
```

It looks like it's due to use of Unix Path separator character in the assertion

https://github.com/eclipse-jkube/jkube/blob/d000d3ffdc1a18047df64f772391d8829f5b3cda/jkube-kit/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceInstallIT.java#L183

### Expected Behavior
HelmServiceInstallIT should pass on windows

### Acceptance Criteria
- [ ] HelmServiceInstallIT should pass on windows

### PR DESCRIPTION
**Pull request by vijaybhagwat24**
## Description
 Test case failure fix
Fixes #3265

## Type of change

 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x ] I have read the [contributing guidelines](https://www.eclipse.dev/jkube/contributing)
 - [x ] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ x] My code follows the style guidelines of this project
 - [x ] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [x ] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [ x] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by manusa** [comment]
### Eclipse JKube [CI Report](https://github.com/eclipse-jkube/ci)

Started new GH workflow run for https://github.com/eclipse-jkube/jkube/pull/3277 (_2024-07-26T03:56:40Z_)

:gear: [JKube E2E Tests (10105447788)](https://github.com/eclipse-jkube/ci/actions/runs/10105447788) :hourglass_flowing_sand:
- :hourglass_flowing_sand: [Bootstrap CI test run (#3277)](https://github.com/eclipse-jkube/ci/actions/runs/10105447788/job/27945940687)

**Comment by vijaybhagwat24** [comment]
test cases:
image

File: jkube-kit/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceInstallIT.java [hunk 1]
```diff
@@ -180,6 +180,6 @@ void whenDependencyReferencedDoesNotExist_thenThrowException() throws IOExceptio
     // When + Then
     assertThatIllegalStateException()
       .isThrownBy(() -> helmService.install(helmConfig))
-      .withMessage(String.format("An error occurred while updating chart dependencies: directory %s/the-dependency not found", tempDir));
+      .withMessage(String.format("An error occurred while updating chart dependencies: directory %s\\the-dependency not found", tempDir));
```
Comment (rohanKanojia) [inline jkube-kit/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceInstallIT.java:183]:
```suggestion
      .withMessage(String.format("An error occurred while updating chart dependencies: directory %s%sthe-dependency not found", tempDir, File.separator));
```

Comment (vijaybhagwat24) [reply jkube-kit/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceInstallIT.java]: Hi Rohan,
corrected and pushed this change.

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**